src/auto-evo/selection_pressure/PredationEffectivenessPressure.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace AutoEvo;
+
+public class PredationEffectivenessPressure : SelectionPressure
+{
+    public readonly Species Prey;
+
+    // Needed for translation extraction
+    // ReSharper disable ArrangeObjectCreationWhenTypeEvident
+    private static readonly LocalizedString NameString = new LocalizedString("MICHE_PREDATION_EFFECTIVENESS_PRESSURE");
+
+    // ReSharper restore ArrangeObjectCreationWhenTypeEvident
+
+    private static readonly Compound Oxytoxy = SimulationParameters.Instance.GetCompound("oxytoxy");
```

Comment:
Registry loaded stuff shouldn't be in static variables.

Reply:
Most of them have static variables because otherwise the MutationStrategies constructors error. Anything that does not need to be marked static is not.

Reply:
~~You can put the field initializer in the constructor.~~ Update: I'm like 99% sure that this works:

```c#
private readonly Compound atp = SimulationParameters.Instance.GetCompound("atp");
```

As used here: https://github.com/Revolutionary-Games/Thrive/blob/3f77b72693693028db228a430462fad2237700f0/src/microbe_stage/organelle_components/MovementComponent.cs#L11

Reply:
I remove the static I just get this: `An object reference is required for the non-static field, method, or property 'AvoidPredationSelectionPressure.ChitinMembrane'`

Reply:
Like a default value or? Or base constructor? For those cases using null as a placeholder value meaning "no value, will be fetched in the method" has been what I do.

Still some other way needs to be used here than allowing a static field to hold a Compound reference.

Reply:
The selection pressures all create mutation strats (with these compounds) and pass them into there parent though the base constructor. I don't see a good way to avoid this being an issue.

Reply:
Some other alternative needs to be made here. Could maybe even make the super class constructors take in internal names of compounds and they then handle calling `GetCompound` if nothing else comes to mind for anyone.

Reply:
That might actually simplify some of the code and fixes this so I will just go ahead and implement that.